R/codecov.R

```diff
@@ -159,6 +159,10 @@ codecov <- function(...,
 
   coverage_json <- to_codecov(coverage)
 
+  if (!quiet) {
```

Comment:
I would prefer not to change this functionality.

Reply:
ok, I have taken this back out